config/500-controller.yaml

```diff
@@ -45,7 +45,7 @@ spec:
           - name: BROKER_INGRESS_IMAGE
             value: github.com/knative/eventing/cmd/broker/ingress
           - name: BROKER_INGRESS_SERVICE_ACCOUNT
-            value: default
+            value: eventing-broker-ingress
```

Comment:
was this actually an oversight, for ingress doing `default` originally ?

Reply:
I had it as `default` originally because it didn't need any permissions, so I didn't see a reason to create a new ServiceAccount for it. In the future, should I create a named ServiceAccount even if it doesn't require any permissions?